gui/dig.lua

```diff
@@ -209,6 +211,47 @@ function GenericOptionsPanel:init()
             show_tooltip = true,
             on_change = self:callback("change_shape"),
         },
+
+        widgets.ResizingPanel { autoarrange_subviews = true, subviews = {
+            widgets.ToggleHotkeyLabel {
+                key = 'CUSTOM_T',
+                view_id = 'transform',
+                label = 'Transform',
+                active = true,
+                enabled = true,
+                initial_option = false,
+                on_change = nil
+            },
+            widgets.ResizingPanel { view_id = 'transform_panel',
+                visible = function() return self.dig_panel.subviews.transform:getOptionValue() end,
+                subviews = {
+                    widgets.HotkeyLabel {
+                        key = 'STRING_A040',
+                        frame = { t = 1, l = 1 }, key_sep = '',
+                        on_activate=self.dig_panel:callback('on_transform', 'ccw'),
+                    },
+                    widgets.HotkeyLabel {
+                        key = 'STRING_A041',
+                        frame = { t = 1, l = 2 }, key_sep = '',
+                        on_activate=self.dig_panel:callback('on_transform', 'cw'),
+                    },
+                    widgets.HotkeyLabel {
+                        key = 'STRING_A095',
+                        frame = { t = 1, l = 3 }, key_sep = '',
+                        on_activate=self.dig_panel:callback('on_transform', 'flipv'),
+                    },
+                    widgets.HotkeyLabel {
+                        key = 'STRING_A061',
+                        frame = { t = 1, l = 4 }, key_sep = ':',
+                        on_activate=self.dig_panel:callback('on_transform', 'fliph'),
+                    },
+                    widgets.WrappedLabel {
+                        frame = { t = 1, l = 7 },
+                        text_to_wrap = function() return 'Transformation directions' end
```

Comment:
this looks like a stub. perhaps the "transform reduction" code in gui/quickfort would be useful to reuse here?

Reply:
Maybe i could put "`(): Rotate clockwise/counter-clockwise`" and `_=: Flip horizontally/vertically` and split them into two lines.